app/core/store/modules/me.coffee

```diff
@@ -17,6 +17,9 @@ module.exports = {
     isAdmin: (state) ->
       permissions = state.permissions or []
       return permissions.indexOf('admin') > -1
+    isLicensor: (state) ->
+      permissions = state.permissions or []
+      return permissions.indexOf('licensor') > -1
```

Comment:
Not a big deal, but you can simplify these as such:
`isLicensor: (state) -> (state?.permissions or []).indexOf('licensor') > -1`